content/webapp/components/ThemeSourcedDescription/index.tsx

```diff
@@ -0,0 +1,142 @@
+import { FunctionComponent, useEffect, useRef, useState } from 'react';
+import styled from 'styled-components';
+import Space from '@weco/common/views/components/styled/Space';
+import { font } from '@weco/common/utils/classnames';
+import WikidataLogo from '@weco/content/components/ThemeSourcedDescription/WikidataLogo';
+import MeshLogo from '@weco/content/components/ThemeSourcedDescription/mesh-logo.png';
+import Image from 'next/image';
+
+const SOURCE_BOX_WIDTH = 224;
+
+const SourcePill = styled(Space).attrs({
+  className: `${font('intr', 6)} source-pill`,
+})`
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+  border-radius: 20px;
+  background-color: ${props => props.theme.color('accent.green')}40;
+  cursor: default;
+  height: 22px;
+  vertical-align: middle;
+
+  &:is(:hover, :focus-within) .source-box-container {
+    opacity: 1;
+    visibility: visible;
+  }
+`;
+
+const Paragraph = styled(Space).attrs({
+  as: 'p',
+  className: font('intr', 3),
+})`
+  display: inline;
+  padding-right: ${props => props.theme.spacingUnits['3']}px;
+
+  &:has(+ .source-pill:focus-within, + .source-pill:hover) {
+    text-decoration: underline;
+    text-decoration-style: dotted;
+  }
+`;
+
+const SourceBoxContainer = styled(Space).attrs({
+  className: 'source-box-container',
```

Comment:
We try to move away from classes as much as possible (we had a lot of unused/unstyled one still lying around!). It can still be necessary based on a condition (like you either have the class or you don't), but otherwise I believe all you need is to reference the name of the Styled components like

```
&:is(:hover, :focus-within) ${SourceBoxContainer} {
    opacity: 1;
    visibility: visible;
  }
```